src/containers/Network/css/nodesTable.scss

```diff
@@ -15,9 +15,12 @@
     letter-spacing: 0px;
     text-align: left;
 
+    tr {
+      border-bottom: 1px solid $gray-800;
```

Comment:
before - we were drawing borders for each cell instead of each row, which was confusing